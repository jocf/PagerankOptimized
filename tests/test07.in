3
0.8
3
A
B
C
2
A B
B C
