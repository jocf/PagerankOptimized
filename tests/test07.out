A 0.0667
B 0.1200
C 0.1627
